src/components/ModelCamera/components/Model.ts

```diff
@@ -0,0 +1,75 @@
+import * as tf from "@tensorflow/tfjs";
+import { Models } from "../../../types/Models";
+import { isMobile } from 'react-device-detect';
+import * as tflite from '@tensorflow/tfjs-tflite'
+// Adds the CPU backend.
+//import '@tensorflow/tfjs-backend-cpu';
+export async function loadModel() {
+    let result: Models = { R_Model: undefined, L_Model: undefined };
+
+    if (isMobile) {
+        tflite.setWasmPath(
+            'https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-tflite@0.0.1-alpha.8/dist/'
+        );
+        console.log('isMobile');
```

Comment:
I think we can remove this console log and the desktop one as well.

Reply:
True. I'll remove it